conans/client/tools/files.py

```diff
@@ -383,3 +383,16 @@ def unix2dos(filepath):
 
 def dos2unix(filepath):
     _replace_with_separator(filepath, "\n")
+
+def rename(src, dst):
+    """
+    rename a file or folder to avoid "Access is denied" error on Windows
+    :param src: Path to the file or folder
+    :param dst: Destination file or folder
+    """
+    if platform.system() == "Windows":
+        retcode = os.system("robocopy /move /e /ndl /nfl".format(src, dst))
```

Comment:
I would recommend subprocess instead, it's lighter and safer than os.system

Reply:
> I would recommend subprocess instead, it's lighter and safer than os.system

Switched to subprocess.